_includes/tutorials/time-concepts/ksql/markup/dev/make-src-file.adoc

```diff
@@ -0,0 +1,5 @@
+Now that you have a series of statements that's doing the right thing, the last step is to put them into a file so that they can be used outside the CLI session. Create a file at `src/statements.sql` with the following content:
```

Comment:
This file isn't used in the harness yaml for time-concepts. Can we remove this?

Reply:
Removed